### ISSUE TITLE
Plugin downloader not working

### ISSUE THREAD

**Issue by felixplasser** [OpenChemistry/avogadroapp#323]
**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: (e.g. 1.93.0)
 - Qt: (e.g., 5.12.8)

**Desktop version: (please complete the following information):**
 - OS: Linux
 - Version 20.04

**Describe the bug**
I tried to activate a plugin via the `Plugin Downloader` feature. But this plugin did not appear in the Avogadro2 menu. It is not clear how one would activate a plugin.

**To Reproduce**
1. Go to `Extensions`
2. Click on `Plugin Downloader`
3. Click on `avogenerators`
4. Click on `Download Selected`.
5. This still looks ok and I get the following output
~~~~
Downloading avogenerators.zip to /home/lunet/cmfp2/.local/share/OpenChemistry/Avogadro
Extracting /home/lunet/cmfp2/.local/share/OpenChemistry/Avogadro/avogenerators.zip to /home/lunet/cmfp2/.local/share/OpenChemistry/Avogadro/inputGenerators/
Extraction successful
~~~~
6. Restart Avogadro
7. I cannot find the Plugin for the input generators anywhere.

**Expected behavior**
The Plugin for the input generators appears somewhere in the menu.

**Additional context**
I tried to search for a solution to this but did not find an answer.

**Comment by ghutchis** [comment]
This is an incredibly old version from Feb. 2020. Please try 1.97, the latest release.
https://github.com/openchemistry/avogadrolibs/releases/latest

For example, you can try the AppImage. I also believe that the DebiChem team has "unstable" packages of 1.96 and 1.97 available.

I don't know why you didn't find a solution - there are several threads on this on the discussion forum: https://discuss.avogadro.cc/

**Comment by ghutchis** [comment]
Can you confirm whether an updated version works? I'd like to close this. Thanks.

**Comment by felixplasser** [comment]
Hi Geoff, sorry I was away ...

I am sure there is a workaround if I try hard enough. So, I'll close the issue.

I guess the feedback here is that a simple `sudo apt install avogadro` no longer provides what I need. This was all I needed for Avogadro 1. But with Avogadro 2 I have to dig deep into the discussion forum to get it to run.

I don't mean to criticise. Avogadro is a great code. But my feedback is that user friendliness went down from version 1 to 2.

Thanks,
Felix

**Comment by ghutchis** [comment]
I can't control what packages are included in the distro - Ubuntu included what was the latest version in early 2020 (1.93). We have, not surprisingly, continued to develop and improve. There have been a lot of bugs fixed in over 2.5 years of development, along with many new features.

As I mentioned, there are newer Debian / Ubuntu packages (in both `testing` and `unstable`), as well as AppImage and Flatpak installs.
https://packages.debian.org/search?keywords=avogadro

I think you'll find that 1.97 has improved .. as will 2.0 when we finish it up.

Thanks!
-Geoff
